### PR TITLE
HTML movie seat booking

### PR DESCRIPTION
**Pull request by kt-std**
## Movie seat booking
[Demo](https://kt-std.github.io/cinema-booking/)
[Code base](https://github.com/kt-std/cinema-booking)

The code is submitted in a dedicated feature branch.
Please, review.

P. S. I've seen some discussions related to the usage of `time` tag for the movie duration. In this [MDN demo](https://interactive-examples.mdn.mozilla.net/pages/tabbed/time.html) duration is being implemented via the `time`. That is why I decided to use it.

### CODE REVIEW

File: submissions/kt-std/html-seat-booking/index.html [hunk 1]
```diff
@@ -0,0 +1,505 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Libre+Barcode+39+Extended+Text&family=Megrim&family=Quicksand:wght@300;500&display=swap" rel="stylesheet"> 
+  </head>
+
+  <body>
+    <main class="movie__container">
+
+      <!-- poster -->
+      <article class="poster__container ">
+        <button class="movie__play-button flex-center" name="button" value="Play trailer"><span class="play"></span></button>
+        <h2 class="poster__heading">ALIEN</h2>
```
Comment (A-Ostrovnyy) [inline submissions/kt-std/html-seat-booking/index.html]: Avoid implementing visual representing in markup. Use CSS for it.

File: submissions/kt-std/html-seat-booking/index.html [hunk 2]
```diff
@@ -0,0 +1,505 @@
...
+          <dt class="info__heading">Genre</dt>
+          <dd class="info__description"><strong>Science fiction</strong>, <b>horror</b></dd>
+          <dt class="info__heading">Starring</dt>
+          <dd class="list__item">
+            <strong class="actor">Tom Skerritt</strong>, 
+            <strong class="actor">Sigourney Weaver</strong>,
+            <strong class="actor">Veronica Cartwright</strong>,
+            <strong class="actor">Harry Dean Stanton</strong>
+          </dd>
+        </dl>
```
Comment (A-Ostrovnyy) [inline submissions/kt-std/html-seat-booking/index.html:41]: 🔥

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
Eye candy view. Extra like for movie choice.)

File: submissions/kt-std/html-seat-booking/index.html (hunk 1 above)
Comment (kt-std) [reply submissions/kt-std/html-seat-booking/index.html]: Firstly I wanted to implement the `article` containing heading, image, and button to make something similar to the poster widget, but then I gave up the idea. Therefore, the heading is not needed anymore. Thank you for your advice! :)

**Comment by kt-std** [comment]
@A-Ostrovnyy, thank you for your time and reviewing! :bow:  Ready to receive further recommendations :rocket:

File: submissions/kt-std/html-seat-booking/css/style.css [hunk 3]
```diff
@@ -0,0 +1,1086 @@
...
+    transform: translate(-50%);
+    color: white;    
+    font-family: var(--space-font);
+    font-weight: var(--thin);
+    font-size: 50px;
+    letter-spacing: 10px;
+    text-shadow: var(--shadow);
+    text-transform: uppercase;
+    animation: textShadow 4s alternate infinite;    
+}
```
Comment (A-Ostrovnyy) [inline submissions/kt-std/html-seat-booking/css/style.css]: This text must be in markup and inside headings tag.
I mean if someone changed the text will it be uppercase?

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
We have some misunderstanding)

File: submissions/kt-std/html-seat-booking/css/style.css (hunk 3 above)
Comment (kt-std) [reply submissions/kt-std/html-seat-booking/css/style.css]: Oops, sorry! :see_no_evil:  I've turned everything back :)

File: submissions/kt-std/html-seat-booking/css/style.css [hunk 4]
```diff
@@ -84,15 +84,12 @@ body {
         url(../assets/poster.jpg);    
 }
 
-.poster__container::before{
-    content: 'Alien';
+.poster__heading{
```
Comment (A-Ostrovnyy) [inline submissions/kt-std/html-seat-booking/css/style.css:87]: `text-transform: uppercase` where are you?)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
)

**Review by A-Ostrovnyy — APPROVED** [review]
My bad, wrong button)
Well done!

**Comment by kt-std** [comment]
@A-Ostrovnyy, warmest thanks! I truly appreciate your support 😊

**Pull request merged**